.env.example

```diff
@@ -21,6 +21,12 @@ LNBITS_RATE_LIMIT_UNIT="minute"
 LNBITS_ALLOWED_IPS=""
 LNBITS_BLOCKED_IPS=""
 
+# Auth Configurations
+SECRET_KEY="" # will default to the superuser value
+ALGORITHM = "HS256"
```

Comment:
Why would you want to make this configurable?

Reply:
I would need at least SECRET_KEY to be configurable so I can put lnbits behind an auth-proxy configured with the same SECRET_KEY for example.

Reply:
Yeah, I was asking about `ALGORITHM` field.